test/Benchmarks/Metrics/ExemplarBenchmarks.cs

```diff
@@ -112,5 +132,18 @@ public void HistogramWithTagReduction()
 
             this.histogramWithTagReduction.Record(random.Next(1000), tags);
         }
+
+        public class LowValueExemplarFilter : ExemplarFilter
```

Comment:
Why not name this `HighValue` as well? It's a bit confusing to use "LowValueFilter" for `HighValueOnly`.